Scripts/Runtime/Jobs/Deferred/JobDeferredNativeArrayForBatchExtension.cs

```diff
@@ -111,10 +116,20 @@ internal static void Initialize()
 #if ENABLE_UNITY_COLLECTIONS_CHECKS
                     JobsUtility.PatchBufferMinMaxRanges(bufferRangePatchData, UnsafeUtility.AddressOf(ref jobData), beginIndex, endIndex - beginIndex);
 #endif
-
                     jobData.Execute(beginIndex, endIndex - beginIndex);
                 }
             }
+
+            private const int DEFAULT_NATIVE_THREAD_INDEX = -1;
```

Comment:
🚲 Consider renaming to UNSET_NATIVE_THREAD_INDEX. Otherwise, it reads a bit like this is a valid thread index.

Reply:
Done